packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-checkbox.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Checkbox, StaticUrl, Text } from '@deriv/components';
-import { isMobile, Jurisdiction } from '@deriv/shared';
+import { is_mobile, Jurisdiction } from '@deriv/shared';
```

Comment:
```suggestion
import { Jurisdiction } from '@deriv/shared';
```

It is not from shared it is coming from ui-store

Reply:
oops